### ISSUE TITLE
Label for notes textarea in checkout form

### ISSUE THREAD

**Issue by amilenkov** [backdrop-contrib/basic_cart#1]
I realized that this module is no longer supported, but I installed it, I tested and I want to share a note, which may be useful in the development of Basic Plus Cart, and other modules.

I maintain a web site of foundation for the visually impaired, where there are good professionals in evaluation to accessibility of sites for the visually impaired. I often ask them to check the sites that I do for accessibility for people with visual impairments.

Today they did a very good assessment of the Basic cart with only one note: There should be a label for notes textarea in checkout form - when the reading program come to this field it says "Edit" and a blind user do not know what it is about.

Overall I am delighted with the Basic cart - there is a huge demand for such a light module for a Web store without online payment. I have been looking for and I've read hundreds of comments on this topic of users of various CMS. I've done several such sites without online payment solution with Uberkart and Drupal Commerce, but they are terribly and unnecessarily complex and heavy for this purpose.

I hope Basic cart will continue it's development as a Basic cart plus, but also not to become complicate unnecessarily and to keep the current simplicity of the system.

I read in another post question about how to make a Top web shop. I do not think that there should be only one Top web shop, but at least two - one for complex and large stores and one for small shops and suitable for small and micro business without online payment solution. I think Basic cart is Top web shop in this sector.

**Comment by biolithic** [comment]
Hi Antony,
Thank you for the message.  I agree that Basic Cart is a good idea when
considering Ubercart and Drupal Commerce which can be too big for a
client's purpose.  If someone wanted the exact port of the Drupal module,
the Basic Cart exists.  Future work (for example, checkboxes for no payment
OR Paypal etc, checkboxes to use Drupal mail OR a transactional email
system like Sendgrid etc) are going into Basic Cart Plus so that there is a
distinction that this is BC, but with additional options and not an exact
port.

**Comment by amilenkov** [comment]
Thank you Andy Martha,

This is a site I am developing now with Backdrop and Basic cart (the 
site is in Bulgarian only):

http://gamaprim.com/prahovi-pozharogasiteli

The theme is my own production and only temporary, the layout also.

I develop site design and theme only after the content is defined and 
its logical structure is clear. On this website we're still working on 
this task. That's why the theme and layout are temporary.

I tried yesterday to install Basic cart plus. What stopped me was the 
requirement to install "form validation scripts". I couldn't figure out 
what it is and how to install. I apologize and I am sorry. I haven't 
been able to figure out what that means.

As a programmer I have quite elementary knowledge of PHP. I suppose, 
however, that Backdrop, in it's philosophy, it shouldn't be very 
difficult for people with my scanty knowledge but still with 
considerable experience in building websites with Drupal.

Regards,

Anthony

**Comment by biolithic** [comment]
Hi Anthony,
Sorry about that.  It is another Backdrop module for including (or not)
Javascript/client-side validation scripts to help the user look up their
address, validate number fields, etc... for the form.

I will include a link tonight after work.  Thanks for trying though!

**Comment by amilenkov** [comment]
Hi Andy,

I will try and use Basic cart plus as soon as I can install it.

Thank you very much!

Antony

On 21-Jun-16 01:39 AM, Andy Martha wrote:

> Hi Anthony,
> Sorry about that.  It is another Backdrop module for including (or not)
> Javascript/client-side validation scripts to help the user look up their
> address, validate number fields, etc... for the form.
> 
> I will include a link tonight after work.  Thanks for trying though!
> 
> On Mon, Jun 20, 2016 at 2:51 PM, Antony Milenkov notifications@github.com
> wrote:
> 
> > Thank you Andy Martha,
> > 
> > This is a site I am developing now with Backdrop and Basic cart (the
> > site is in Bulgarian only):
> > 
> > http://gamaprim.com/prahovi-pozharogasiteli
> > 
> > The theme is my own production and only temporary, the layout also.
> > 
> > I develop site design and theme only after the content is defined and
> > its logical structure is clear. On this website we're still working on
> > this task. That's why the theme and layout are temporary.
> > 
> > I tried yesterday to install Basic cart plus. What stopped me was the
> > requirement to install "form validation scripts". I couldn't figure out
> > what it is and how to install. I apologize and I am sorry. I haven't
> > been able to figure out what that means.
> > 
> > As a programmer I have quite elementary knowledge of PHP. I suppose,
> > however, that Backdrop, in it's philosophy, it shouldn't be very
> > difficult for people with my scanty knowledge but still with
> > considerable experience in building websites with Drupal.
> > 
> > Regards,
> > 
> > Anthony
> > 
> > On 20-Jun-16 19:18 PM, Andy Martha wrote:
> > 
> > > Hi Antony,
> > > Thank you for the message. I agree that Basic Cart is a good idea when
> > > considering Ubercart and Drupal Commerce which can be too big for a
> > > client's purpose. If someone wanted the exact port of the Drupal module,
> > > the Basic Cart exists. Future work (for example, checkboxes for no
> > > payment
> > > OR Paypal etc, checkboxes to use Drupal mail OR a transactional email
> > > system like Sendgrid etc) are going into Basic Cart Plus so that there
> > > is a
> > > distinction that this is BC, but with additional options and not an exact
> > > port.
> > > 
> > > On Mon, Jun 20, 2016 at 4:47 AM, Antony Milenkov <
> > > notifications@github.com>
> > > wrote:
> > > 
> > > > I realized that this module is no longer supported, but I installed it,
> > > > I
> > > > tested and I want to share a note, which may be useful in the
> > > > development
> > > > of Basic Plus Cart, and other modules.
> > > > 
> > > > I maintain a web site of foundation for the visually impaired, where
> > > > there
> > > > are good professionals in evaluation to accessibility of sites for the
> > > > visually impaired. I often ask them to check the sites that I do for
> > > > accessibility for people with visual impairments.
> > > > 
> > > > Today they did a very good assessment of the Basic cart with only one
> > > > note: There should be label for notes textarea in checkout form - when
> > > > the
> > > > reading program come to this field it says "Edit" and a blind user do
> > > > not
> > > > know what it is about.
> > > > 
> > > > Overall I am delighted with the Basic cart - there is a huge demand for
> > > > such a light module for a Web store without online payment. I have been
> > > > looking for and I've read hundreds of comments on this topic of users of
> > > > various CMS. I've done several such sites without online payment
> > > > solution
> > > > with Uberkart and Drupal Commerce, but they are terribly and
> > > > unnecessarily
> > > > complex and heavy for this purpose.
> > > > 
> > > > I hope Basic cart will continue it's development as a Basic cart plus,
> > > > but
> > > > also not to become complicate unnecessarily and to keep the current
> > > > simplicity of the system.
> > > > 
> > > > I read in another post question about how to make a Top web shop. I do
> > > > not
> > > > think that there should be only one Top web shop, but at least two - one
> > > > for complex and large stores and one for small shops and suitable for
> > > > small
> > > > and micro business without online payment solution. I think Basic cart
> > > > is
> > > > Top web shop in this sector.
> > 
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub
> > https://github.com/backdrop-contrib/basic_cart/issues/1#issuecomment-227249886,
> > or mute the thread
> > https://github.com/notifications/unsubscribe/ABJ8DhUp-2qrdMyIrx2XmGlMRUh2vcE_ks5qNu82gaJpZM4I5iE0
> > .
> 
> ---
> 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/backdrop-contrib/basic_cart/issues/1#issuecomment-227291474

## 

Поздрави,

Антони Миленков

---

моб.: 087 777 05 31
тел.: 02 825 72 77
e-mail: antony@milenkov.net
website: www.milenkov.net

**Comment by robertgarrigos** [comment]
Hi @biolithic and @amilenkov, it's been a long time, but I'm able to keep on maintaining this module. I made a PR for this issue. Would you like to give it a try?
@biolithic would you be able to review issues here? Thanks.

**Comment by amilenkov** [comment]
Hi @robertgarrigos
This is good news that seems to come just in time. I am about to develop a new website that will sell only one product. I will install and test Basic Cart in the coming days and share here how it works and if I have any suggestions.

I've made good progress with Ubеrcart ob Backdrop in the meantime and I'm very happy with the attention the BD community is giving to Ubеrcart, but having as option an easier e-selling system like Basic Cart would be very helpful.
